nifi-nar-bundles/nifi-hubspot-bundle/nifi-hubspot-processors/src/main/java/org/apache/nifi/processors/hubspot/GetHubSpot.java

```diff
@@ -82,6 +84,7 @@
         " objects can be set in the 'Result Limit' property.")
 @DefaultSettings(yieldDuration = "10 sec")
 @DefaultSchedule(strategy = SchedulingStrategy.TIMER_DRIVEN, period = "1 min")
+@WritesAttribute(attribute = "mime.type", description = "Sets the mime type to application/json")
```

Comment:
Minor wording adjustment, MIME should be uppercased.
```suggestion
@WritesAttribute(attribute = "mime.type", description = "Sets the MIME type to application/json")
```